.github/workflows/ci.yaml

```diff
@@ -33,7 +33,8 @@ jobs:
         name: Unit & integration tests
         runs-on: "ubuntu-22.04"
         timeout-minutes: 15
-
+        env:
+            COMPOSER_ROOT_VERSION=dev-ibx-8534-core-v5-p2
```

Comment:
Please include in the PR description clear and visible message that the tmp commit needs to be dropped before merging.